pangeo-deploy/values.yaml

```diff
@@ -37,10 +37,12 @@ pangeo:
         enabled: false
 
     scheduling:
-        userScheduler:
-          enabled: false
-        userPlaceholder:
-          enabled: false
+      userScheduler:
```

Comment:
Can you describe the effect of these changes or link to an issue?

Reply:
check this out: https://zero-to-jupyterhub.readthedocs.io/en/latest/optimization.html?highlight=scheduling